### PR TITLE
Refactor/project part 3

### PR DESCRIPTION
**Pull request by sazanik**

### CODE REVIEW

File: src/screens/AuthorsList/AuthorsList.tsx [hunk 1]
```diff
@@ -140,8 +92,14 @@ export const AuthorsList = (): JSX.Element => {
 
   return (
     <Box sx={styles.box}>
-      {!authors.length ? (
-        <Button onClick={clickHandler} aria-label='add' sx={styles.button}>
+      {!authors.length && isGeneralLoading ? (
```
Comment (ArturRusak) [inline src/screens/AuthorsList/AuthorsList.tsx:95]: Do you use somewere other loader ?

File: src/screens/AuthorsList/AuthorsList.tsx [hunk 2]
```diff
@@ -140,8 +92,14 @@ export const AuthorsList = (): JSX.Element => {
...
+        <Button
+          onClick={() => {
+            setIsEdit(false);
```
Comment (ArturRusak) [inline src/screens/AuthorsList/AuthorsList.tsx:98]: for me better to move in handler

File: src/screens/AuthorsList/AuthorsList.tsx [hunk 3]
```diff
@@ -37,88 +38,39 @@ export const AuthorsList = (): JSX.Element => {
...
-        setIsOpenModal(true);
-        break;
+  const openAuthor = (params: GridRenderCellParams): void => {
+    navigate(`/authors/${params.id}`);
+  };
 
-      case Actions.DELETE:
-        setIsOpenDialog(true);
-        break;
+  const renderNameCells = (params: GridRenderCellParams): JSX.Element => (
```
Comment (ArturRusak) [inline src/screens/AuthorsList/AuthorsList.tsx:45]: use destructure here

**Review by ArturRusak — APPROVED** [review]

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 1 above)
Comment (sazanik) [reply src/screens/AuthorsList/AuthorsList.tsx:95]: No, i am not

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 3 above)
Comment (sazanik) [reply src/screens/AuthorsList/AuthorsList.tsx:45]: `  const renderNameCells = (params: GridRenderCellParams): JSX.Element => (
    <Button sx={styles.buttonLeft} onClick={() => openAuthor(params)}>
      {params.value}
    </Button>
  );`

Here params is used twice
openAuthor(params)}
{params.value}

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 2 above)
Comment (sazanik) [reply src/screens/AuthorsList/AuthorsList.tsx:98]: fixed

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 3 above)
Comment (sazanik) [reply src/screens/AuthorsList/AuthorsList.tsx:45]: fixed

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 1 above)
Comment (ArturRusak) [reply src/screens/AuthorsList/AuthorsList.tsx:104]: I propose to use just `Loading` name

File: src/screens/BooksList/BookList.tsx [hunk 4]
```diff
@@ -34,18 +34,23 @@ export const BooksList = (): JSX.Element => {
...
+    navigate(`/books/${event.id}`);
+  };
+
+  const handlerClickAddBook = (): void => {
+    setIsEdit(false);
+    setIsOpenModal(true);
+  };
+
+  const cellClickHandler = ({ field, row }: GridCellParams): void => {
+    if (field !== Fields.EDITING) return;
```
Comment (ArturRusak) [inline src/screens/BooksList/BookList.tsx:47]: code style!!!!!!!!!!!!!!

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 3 above)
Comment (ArturRusak) [reply src/screens/AuthorsList/AuthorsList.tsx]: Just didn't see you use params for function. 
We could keep your solution in the future.

File: src/screens/AuthorsList/AuthorsList.tsx (hunk 1 above)
Comment (sazanik) [reply src/screens/AuthorsList/AuthorsList.tsx:104]: Sorry, i understand you. Name _loading_ is using to each store slice

File: src/screens/BooksList/BookList.tsx (hunk 4 above)
Comment (sazanik) [reply src/screens/BooksList/BookList.tsx:47]: fixed

**Pull request merged**